sources/org.osbuild.containers-storage

```diff
@@ -87,24 +87,22 @@ class ContainersStorageSource(sources.SourceService):
     def fetch_all(self, items) -> None:
         return
 
-    def exists(self, checksum, desc) -> bool:
-        image_id = checksum
-        image = desc["image"]
-        imagename = image["name"]
-        digest = image["digest"]
-
-        source = self.local_image_name(imagename)
-        res = sp.run(["skopeo", "inspect", "--raw", "--config", source], check=False, capture_output=True)
+    def exists(self, checksum, _) -> bool:
+        source = self.local_image_name(checksum)
```

Comment:
This commit breaks things (at least for me). I think we need to split the checksum here. Or am I missing something?

Reply:
You're right.  We should fix the test to include it too because this value should be coming from the manifest and include the algo prefix.